hack/vendor.sh

```diff
@@ -18,7 +18,7 @@ clone git github.com/microsoft/hcsshim 7f646aa6b26bcf90caee91e93cde4a80d0d8a83e
 clone git github.com/mistifyio/go-zfs v2.1.1
 clone git github.com/tchap/go-patricia v2.1.0
 clone git golang.org/x/net 3cffabab72adf04f8e3b01c5baf775361837b5fe https://github.com/golang/net.git
-clone hg code.google.com/p/gosqlite 74691fb6f837
+clone git github.com/mxk/go-sqlite 167da9432e1f4602e95ea67b67051cfa34412e3f
```

Comment:
I'm tryinng that sha tho

On Thu, Sep 3, 2015 at 5:19 PM, Jessie Frazelle <jess@docker.com> wrote:

> https://code.google.com/p/go-sqlite/
>
> On Thu, Sep 3, 2015 at 5:18 PM, Tianon Gravi <notifications@github.com>
> wrote:
>
>> In hack/vendor.sh
>> <https://github.com/docker/docker/pull/16054#discussion_r38711350>:
>>
>> > @@ -18,7 +18,7 @@ clone git github.com/microsoft/hcsshim 7f646aa6b26bcf90caee91e93cde4a80d0d8a83e
>> >  clone git github.com/mistifyio/go-zfs v2.1.1
>> >  clone git github.com/tchap/go-patricia v2.1.0
>> >  clone git golang.org/x/net 3cffabab72adf04f8e3b01c5baf775361837b5fe https://github.com/golang/net.git
>> > -clone hg code.google.com/p/gosqlite 74691fb6f837
>> > +clone git github.com/mxk/go-sqlite 167da9432e1f4602e95ea67b67051cfa34412e3f
>>
>> Do you happen to have a link to where they suggest using this particular
>> repo? I can't find any suggestion like that myself. [image: :cry:]
>>
>> Alternatively, we could switch over to go-sqlite3 completely (#15387
>> (comment)
>> <https://github.com/docker/docker/pull/15387#issuecomment-137606204>).
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/docker/docker/pull/16054/files#r38711350>.
>>
>
>